syntax = "proto3";

package  hnc_rpc;

message HncRpcHeader {
    bytes server_name = 1;
    bytes method_name = 2;
    uint32 args_size = 3;
};

